Source/control.cpp

```diff
@@ -22,23 +23,15 @@
 #include "towners.h"
 #include "trigs.h"
 #include "utils/language.h"
+#include "controls/keymapper.hpp"
 
 namespace devilution {
 namespace {
+
 CelOutputBuffer pBtmBuff;
 CelOutputBuffer pLifeBuff;
 CelOutputBuffer pManaBuff;
-std::optional<CelSprite> pTalkBtns;

```

Comment:
Where did these go?

Reply:
Bad rebase. All those refactorings make rebasing a bit difficult sometimes. Fixing.

Reply:
Yeah sorry about all the churn